### PR TITLE
Replace Share block ToggleControl components with CheckboxControl components

### PR DESCRIPTION
**Pull request by paranoia1906**

### CODE REVIEW

**Comment by paranoia1906** [comment]
https://github.com/godaddy/coblocks/issues/607

File: src/blocks/share/block.json [hunk 1]
```diff
@@ -0,0 +1,76 @@
+{
+    "name": "social",
+    "category": "frontend",
```
Comment (richtabor) [inline src/blocks/share/block.json]: This should be `coblocks`

File: src/blocks/share/editor.scss [hunk 2]
```diff
@@ -94,3 +94,13 @@
 .components-range-control + .components-coblocks-inspector__social-button-size {
 	margin-top: -13px !important;
 }
+
+.components-share-item-attributes__label {
```
Comment (richtabor) [inline src/blocks/share/editor.scss]: Let's change this class to `.components-social-icons-list__label`.

File: src/blocks/share/inspector.js [hunk 3]
```diff
@@ -150,53 +150,51 @@ class Inspector extends Component {
 								className="components-coblocks-inspector__social-button-size"
 							/>
 						}
+						<div className="components-share-item-attributes">
```
Comment (richtabor) [inline src/blocks/share/inspector.js]: Let's change this class to `.components-social-icons-list`

File: src/blocks/share/inspector.js [hunk 4]
```diff
@@ -150,53 +150,51 @@ class Inspector extends Component {
...
+							<p className="components-share-item-attributes__label">{__('Icons')}</p>
```
Comment (richtabor) [inline src/blocks/share/inspector.js]: Let's change this class to `.components-social-icons-list__label`

**Comment by paranoia1906** [comment]
ESlint and minor changes applied.

**Comment by paranoia1906** [comment]
Pull now also handles https://github.com/godaddy/coblocks/issues/621

**Comment by paranoia1906** [comment]
Made some commits on this branch by mistake. Its reverted to match Rich's commit 
```
ESLint + add save
5534fab
```

**Comment by richtabor** [comment]
Good to go ✅

**Pull request merged**